zconf.h.in

```diff
@@ -127,7 +127,7 @@ typedef void const *voidpc;
 typedef void       *voidpf;
 typedef void       *voidp;
 
-typedef uint32_t z_crc_t;
+typedef unsigned int z_crc_t; /* Don't use uint32_t */
```

Comment:
Consider removing comment and changing the comment above from:
`/* Fallback for something that includes us */` to:
`/* Legacy zlib typedefs for backwards compatibility, don't assume stdint.h is defined */`